File: bench/fib45.c
int printf(const char *fmt, ...);

long fib(long n) {
    if (n < 2) {
        return n;
    }
    else {
        return fib(n-2) + fib(n-1);
    }
}

int main() {
    printf("%li\n", fib(45));
    return 0;
}

File: bench/erlang/fib45.erl
#!/usr/bin/env escript

-module('fib-recurs').
-export([main/1]).
-mode(compile).

-spec fib(int) -> int.
fib(N) when N < 2 ->
    N;
fib(N) ->
    fib(N - 2) + fib(N - 1).

main(_) ->
    io:format("~p~n", [fib(45)]).